pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -36,6 +36,21 @@ in {
     };
   };
 
+  elasticsearch_cloud_aws = esPlugin rec {
+    name = "elasticsearch-cloud-aws-${version}";
+    pluginName = "elasticsearch-cloud-aws";
+    version = "2.4.2";
+    src = fetchurl {
+      url = "https://github.com/elastic/elasticsearch-cloud-aws/archive/v2.4.2.zip";
```

Comment:
This should probably be `fetchFromGitHub`. It uses `fetchzip` behind the scenes, which hashes the contents of the archive rather than the archive file itself, due to GitHub often producing different archive hashes for the same contents over time (timestamps and other irrelevant metadata).